### ISSUE TITLE
AttributeError: 'Options' object has no attribute 'get_all_related_objects'

### ISSUE THREAD

**Issue by FrancescoSaverioZuppichini** [bruth/django-preserialize#19]
Hello,

I get this error:

```
AttributeError: 'Options' object has no attribute 'get_all_related_objects'
```

This is my code:

```python
  user  = User.objects.get(pk=user_id)
   print(serialize(user))
```

```python
class User(models.Model):
    displayName = models.CharField(max_length=28)
    email = models.CharField(max_length=28)
    isNearby = models.BooleanField(default=False)

```
Any Idea?

**Comment by bruth** [comment]
What version of Django are you using?

**Comment by FrancescoSaverioZuppichini** [comment]
latest one

**Comment by bruth** [comment]
Ah, this package has been tested up to 1.10. They must have changed something in 1.11.

**Comment by FrancescoSaverioZuppichini** [comment]
My version:

(1, 10, 5, 'final', 0)

**Comment by bruth** [comment]
I never made a release with the new version! Just released 1.2.1.

**Comment by FrancescoSaverioZuppichini** [comment]
Nice bro.
